MetricsReloaded/metrics/pairwise_measures.py

```
@@ -776,33 +878,29 @@ def negative_predictive_values(self):
-    #         print("Already correct value for fbeta option")
-    #     return self.fbeta()
+        return self.tn() / (self.fn() + self.tn() + self.smooth_dr)
+
+    def dice_score(self):
+        """
+        This function returns the dice score coefficient between a reference
+        and prediction images
+
+        :return: dice score
+        """
+        if not "fbeta" in self.dict_args.keys():
```

Ruff (0.16.0)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)